pkg/network/http/ebpf_main.go

```diff
@@ -144,15 +144,28 @@ func newEBPFProgram(c *config.Config, offsets []manager.ConstantEditor, sockFD *
 		},
 	}
 
-	sslProgram, _ := newSSLProgram(c, sockFD)
-	goTLSProgram, _ := NewGoTLSProgram(c)
+	var ebpfSubprograms []subprogram
+	sslProgram, err := newSSLProgram(c, sockFD)
+	if err != nil {
+		log.Errorf("failed creating ssl ebpf subprogram: %w", err)
+	} else {
+		ebpfSubprograms = append(ebpfSubprograms, sslProgram)
+	}
+
+	goTLSProgram, err := newGoTLSProgram(c)
+	if err != nil {
+		log.Errorf("failed creating go tls ebpf subprogram: %w", err)
+	} else {
+		ebpfSubprograms = append(ebpfSubprograms, goTLSProgram)
+	}
+
```

Comment:
still make no sense for me
both `newSSLProgram` and `newGoTLSProgram` will never return an error

Reply:
fixed